### ISSUE TITLE
Incorrect mimetype breaks build

### ISSUE THREAD

**Issue by melindmi** [graphology/graphology#489]
Hi,
When building an old project I get this error. It used to work well previously.
Anyone knows what the issue could be ?

Module not found: Error: Can't resolve 'obliterator/iterator' in '/path_to_modules/node_modules/graphology/dist'
Did you mean 'iterator.js'?
BREAKING CHANGE: The request 'obliterator/iterator' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

**Comment by Yomguithereal** [comment]
> Anyone knows what the issue could be ?

Yes, I tried to adapt so that ESM-first system can work out-of-the-box with graphology but because of the packaging schism it does not seem to work that well... Can you report what building tool you are using?

Fixing the `graphology` dependency to `0.25.0` or `0.25.1` should be a good workaround in your case for now.

**Comment by Yomguithereal** [comment]
Do you have the same issue with `0.25.0`? If so, the problem lies with `react-scripts`, which is known to do weird stuff wrt packaging and not `graphology` itself. Re-reading your error message it seems to be a configuration issue on the `react-scripts` end, namely regarding its choice of extension resolving and module type choice.

**Comment by melindmi** [comment]
Same issue with 0.25.0 indeed.

**Comment by matedo1** [comment]
Hi @Yomguithereal 

We had the same issue on our project after `graphology` new releases were recently done, we also use `"react-scripts": "5.0.1"`. I can confirm downgrading `graphology` from `0.25.3` to `0.25.1` as workaround solution fixed it for now. 

Unfortunately we are not able to use latest `graphology` releases at this moment.

**Comment by melindmi** [comment]
The workaround I found was to replace react-scripts to craco and add the configuration as described here: [https://stackoverflow.com/questions/70964723/webpack-5-in-ceate-react-app-cant-resolve-not-fully-specified-routes/75109686#75109686](url)

**Comment by Yomguithereal** [comment]
I published v0.25.4 reverting back to v0.25.1 package semantics to avoid half-baked ESM-first support. This should fix most issues but we still don't have ESM-first support which is too much of a hassle currently.
